Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -460,10 +460,11 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .AddParameter("objectvar", _("Variable"));
 
   obj.AddCondition("ObjectVariableChildExists",
-                   _("Child existence"),
-                   _("Check if the specified child of the variable exists."),
+                   _("Child existence of an object structure variable"),
+                   _("Check if the specified child of the object "
+                     "structure variable exists."),
```

Comment:
To me, the full name should allow users to scan a list. It should be as short as possible. It can be a bit imprecise as long as it give the general idea. The description is here to add necessary details to fully understand the instruction.
I prefer the old name.